README.rst

```diff
@@ -84,20 +90,36 @@ Launch Sphinx in an ephemeral pex environment using the Sphinx entry point ``sph
 
     $ pex sphinx -e sphinx:main -- --help
 
-Build a standalone pex binary into ``pex.pex`` using the ``pex`` console_scripts entry point:
+Using Entry Points
+~~~~~~~~~~~~~~~~~~
+
+Projects specifying a ``console_scripts`` entry point (with
```

Comment:
The `with` implies Pex only works with those three. It also works with PDM or any other PEP-517/PEP-51 build system.

I think you really have to drop the parenthese and take a few sentences to say something like "Examples of how a console script is configured in a projects build can be found here..". But I really think, at that point, with all the misleading bits finally mopped up, you've lost the thread and drifted pretty far off base. I'm very much not convinced it makes sense to re-document like this on behalf of what's effectively required required reading / prerequisite knowledge. Perhaps just stick to your first link and use lead-in language like "One way these are specified is described here: ....".

Reply:
Nah, I'll just remove it.